Code/GraphMol/ForceFieldHelpers/ANI/ANI.h

```diff
@@ -0,0 +1,61 @@
+#include <RDGeneral/export.h>
+#ifndef RD_ANICONVENIENCE_H_
+#define RD_ANICONVENIENCE_H_
+
+#include <ForceField/ForceField.h>
+#include <GraphMol/ForceFieldHelpers/FFConvenience.h>
+#include "Builder.h"
+
+namespace RDKit {
+class ROMol;
+namespace ANI {
+//! Convenience function for optimizing a molecule using ANI
+/*
+  \param mol          the molecule to use
+  \param maxIters     the maximum number of force-field iterations
+  \param modelType    Name of ANI style model (ANI-1x or ANI-1ccx)
+  \param ensembleSize Number of Neural Networks inside the model
+  \param confId       the optional conformer id, if this isn't provided, the
+  molecule's default confId will be used.
+
+  \return a pair with:
+     first: 0 if the optimization converged, 1 if more iterations are required.
+     second: the energy
+*/
+std::pair<int, double> ANIOptimizeMolecule(ROMol &mol, std::string modelType,
+                                           unsigned int ensembleSize,
+                                           int confId = -1,
+                                           int maxIters = 1000) {
+  ForceFields::ForceField *ff =
+      ANI::constructForceField(mol, modelType, ensembleSize, confId);
```

Comment:
It would be nicer/safer here, and in the other convenience functions, to use a unique_ptr:
```suggestion
  std::unique_ptr<ForceFields::ForceField> ff(ANI::constructForceField(mol, modelType, ensembleSize, confId));
```